### PR TITLE
data: marameref

### PR DESCRIPTION
**Pull request by marameref**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/marameref.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Amarachi Omereife",
+  "bio": "Seasoned Software Engineer and Azure Cloud Engineer **Proud Techie**. Health & Wellness Enthusiast"
```
Comment (sital002) [inline data/marameref.json:3]:
```suggestion
  "bio": "Seasoned Software Engineer and Azure Cloud Engineer **Proud Techie**. Health & Wellness Enthusiast",
```
There is a comma missing here.

File: data/marameref.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/marameref",
+      "icon": "marameref"
+    }
+    {
+      "name": "Follow me on linkedin",
+      "url": "https://www.linkedin.com/in/amarachi-crystal-omereife-416a081a1/",
+      "icon": "Amarachi Crystal Omereife"
+    }
```
Comment (sital002) [inline data/marameref.json:14]:
```suggestion
    },
```

File: data/marameref.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/marameref",
+      "icon": "marameref"
+    } 
+  ]
```
Comment (sital002) [inline data/marameref.json:20]:
```suggestion
  ],
```

File: data/marameref.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+  "Links": [
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/marameref",
+      "icon": "marameref"
+    }
```
Comment (sital002) [inline data/marameref.json:27]:
```suggestion
    },
```

File: data/marameref.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+    {
+      "name": "Follow me on Medium",
+      "url": "https://medium.com/@marameref",
+      "icon": "Amara Crystal Omereife"
+    }
+}
```
Comment (sital002) [inline data/marameref.json:34]:
```suggestion
]
}
```

File: data/marameref.json [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+      "icon": "Amarachi Crystal Omereife"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/marameref",
+      "icon": "marameref"
+    } 
+  ],
+
+  "Links": [
```
Comment (sital002) [inline data/marameref.json:22]:
```suggestion
  "links": [
```
It should be links instead of Links

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**